src/fairchem/core/models/equiformer_v2/equiformer_v2.py

```diff
@@ -636,7 +636,13 @@ def forward(self, data: Batch, emb: dict[str, torch.Tensor | GraphData]):
             dtype=node_energy.dtype,
         )
         energy.index_add_(0, data.batch, node_energy.view(-1))
-        return {"energy": energy / self.avg_num_nodes}
+        if self.agg_fn == "sum":
+            return {"energy": energy / self.avg_num_nodes}
```

Comment:
One other common aggregation function used is max. We could thrown that in as well.

Reply:
i'll leave it to when someone need to use it? this reduction is a bit custom since our sum is not really a "sum", dont want to make it seem too generic